packages/tasks/src/model-libraries.ts

```diff
@@ -650,6 +650,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		docsUrl: "https://github.com/jasonppy/VoiceCraft",
 		snippets: snippets.voicecraft,
 	},
+	yolov10: {
```

Comment:
Nice! Note that this might impact few dozen repos https://huggingface.co/models?other=yolov10

Reply:
This also impacts quants/ conversions as well.